src/main/resources/templates/user.html

```diff
@@ -147,9 +129,29 @@ <h1>Welcome to The Amazing Online Bookstore</h1>
   </tbody>
 </table>
 
-<form action="#" th:action="@{/user/{username}/search(username=${username})}" method="get">
+<label>Sort By:</label>
+<select id="sortOption">
+  <option value="ISBN">ISBN</option>
+  <option value="Title">Title</option>
+  <option value="Author">Author</option>
+  <option value="Description">Description</option>
+  <option value="Publisher">Publisher</option>
+  <option value="Stock">Stock</option>
+</select>
+<button type="button" id="sortButton" onclick="sortTable()">Select option</button>
+
+<br>
+<br>
+
+<label>Filter By: </label>
+<input type="text" id="searchedItem" placeholder="Search author or publisher..." onkeyup="showOption()"/>
+
+<br>
+<br>
+
+<!-- <form action="#" th:action="@{/user/{username}/search(username=${username})}" method="get">
```

Comment:
Looks good, we'll have to have some graceful degradation in here in the future. But the SPA aspect is working great